app/controllers/application_controller.rb

```diff
@@ -5,6 +5,16 @@ class ApplicationController < ActionController::Base
   responders :flash
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
+  helper do
```

Comment:
@timurvafin i think solution with a helper module is a better approach. The controller would remain smaller and clearer.